www/js/recommendations.js

```diff
@@ -12,15 +12,42 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
 
     $scope.name = "Cannot Retrieve Suggestion";
     $scope.mode = "Cannot Retrieve Mode";
+    $scope.yelp = {
+      "headers": {
+    		"Authorization": "Bearer your-api-key",
```

Comment:
I'm pretty sure that the `your-api-key` here needs to be replaced with an actual API key

Reply:
And given `yelpfusion.json`, do you even need this here?